tap_hubspot/streams.py

```diff
@@ -69,6 +68,22 @@ def schema(self) -> dict:
             self.cached_schema, self.properties = self.get_custom_schema()
         return self.cached_schema
 
+class ArchivedCompaniesStream(CompaniesStream):
+    """Define custom stream."""
+    name = "archived_companies"
+    schema_filepath = ""
+
+    def get_url_params(self, context: Optional[dict], next_page_token: Optional[Any]) -> Dict[str, Any]:
+        params = super().get_url_params(context, next_page_token)
+        params['archived'] = True
```

Comment:
Why are we not adding `params['properties'] = ','.join(self.properties)` like for the regular CompaniesStream?